sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveSparkSubmitSuite.scala

```diff
@@ -19,7 +19,7 @@ package org.apache.spark.sql.hive
 
 import java.io.{BufferedWriter, File, FileWriter}
 
-import scala.tools.nsc.Properties
+import scala.util.Properties
```

Comment:
This works in 2.11 and 2.12?

Reply:
Yes. And

`scala.util.Properties` is better than `scala.tools.nsc.Properties` !

Reply:
`scala.tools` is part of scala-compiler